src/Jackett.Common/Indexers/Nordicbits.cs

```diff
@@ -462,30 +462,22 @@ protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuer
         /// <param name="url">Search url for provider</param>
         /// <param name="page">Page number to request</param>
         /// <returns>URL to query for parsing and processing results</returns>
-        private string BuildQuery(string term, TorznabQuery query, string url, int page = 0)
+        private string BuildQuery(string searchTerm, TorznabQuery query, string url, int page = 0)
         {
-            var parameters = new NameValueCollection();
+            var qc = new NameValueCollection();
```

Comment:
move lower line into object initializer.
```suggestion
            var qc = new NameValueCollection
            {
                {"incldead", "1"}
            };
```

Reply:
fixed